examples/simple/src/tags/TagList.tsx

```diff
@@ -75,18 +76,17 @@ const SubTree = ({ level, root, getChildNodes, openChildren, toggleNode }) => {
     const open = openChildren.includes(root.id);
     return (
         <Fragment>
-            <ListItem
-                onClick={() => hasChildren && toggleNode(root)}
-                style={{ paddingLeft: level * 16 }}
-            >
-                {hasChildren && open && <ExpandLess />}
-                {hasChildren && !open && <ExpandMore />}
-                {!hasChildren && <div style={{ width: 24 }}>&nbsp;</div>}
-                <ListItemText primary={root.name.en} />
+            <ListItem style={{ paddingLeft: level * 16 }}>
```

Comment:
Don't you need `disablePadding` here too?

Reply:
Without `disablePadding` 
![image](https://github.com/user-attachments/assets/731866ec-c0bd-4b2b-8322-84096e3c3666)

With `disablePadding`
![image](https://github.com/user-attachments/assets/23f7edb0-7132-4991-a7dd-c27dc37c5370)